### PR TITLE
chore: update publish action

### PR DESCRIPTION
**Pull request by kurkle**
This won't solve the current problem with release action, but the next one that would be faced.

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -11,10 +11,10 @@ jobs:
     if: github.repository_owner == 'chartjs'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
-          node-version: 12
+          node-version: current
```
Comment (simonbrunel) [inline .github/workflows/publish.yml:17]: Do we actually still need to specify a version?

Comment (kurkle) [reply .github/workflows/publish.yml:17]: Probably would be ok to just not specify a version

**Comment by simonbrunel** [comment]
> This won't solve the current problem with release action, but the next one that would be faced.

@kurkle Out of curiosity, what's the next problem that this PR would avoid?

**Review by simonbrunel — APPROVED** [review]

**Comment by simonbrunel** [comment]
FYI: when an [innocent environment variable](https://github.com/chartjs/chartjs-plugin-datalabels/commit/61e38903333a3da882f1b64dac14af16e8c52d17) breaks your workflow 😅

**Pull request merged**